artemis-server/src/main/java/org/apache/activemq/artemis/core/remoting/impl/netty/HttpAcceptorHandler.java

```diff
@@ -175,7 +176,8 @@ private void piggyBackResponses(ByteBuf buf) {
                } catch (InterruptedException e) {
                   break;
                }
-            } while (responses.isEmpty());
+            }
```

Comment:
everything else is fine... just this while breaking line... unless we agree on change the rules...